index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css">
+    <link href="https://fonts.googleapis.com/css2?family=Cairo:wght@200&family=Gudea:wght@400;700&family=IBM+Plex+Sans:wght@300;400;500&family=Lato:wght@300;400;700&family=Open+Sans:wght@300;400;500&family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <title>Document</title>
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSK2rfCGbrDopSuC8R5mnnRymSX0r1mVkqQAEiwNh6mHMDNfQ4NTHlySOnrq7B_XZEjhlU&usqp=CAU" id="header-img">
+        <nav id="nav-bar">
+      
```

Comment:
- [ ] Kindly make sure to remove this empty line as it makes your file not clean.